build.gradle

```diff
@@ -0,0 +1,27 @@
+plugins {
+    id 'com.indeed.oss' version '1+'
+    id 'maven-publish'
+    id 'java'
+}
+
+indeedOss.activateFeature 'library'
+indeedLibrary.name = 'proctor-parent'
+
+publishing {
+    publications {
+        mavenJavaParent(MavenPublication) {
+            groupId = 'com.indeed'
+            artifactId = 'proctor-parent' // use the name from ext['indeed.publish.name']
+            version = '9999-SNAPSHOT' // use whatever version makes sense for your project
+
+            from components.java
+        }
+    }
+}
+
+task linkLocalPublish(type: Exec) {
+    commandLine './symlinkproctor.sh'
+}
```

Comment:
`dependsOn 'publishToMavenLocal'` inside this block

then remove the line below